content/md/permissions-settings/access-rights-on-data-quality.md

```diff
@@ -0,0 +1,28 @@
+---
+id: access-rights-on-data-quality
+themes: permissions-settings
+title: Set rights on **data quality**
+popular: false
+ee-only: true
+related: understand-data-quality, improve-data-quality, manage-data-quality
+---
+
+# Overview
+You can easily restrict the activation and deactivation of data quality insights on attribute groups depending on the user groups your users are in.
+
+By default, all users with the right to edit an attribute group, have the right to activate and deactivate data quality insights.
+
+# Set rights to activate and deactivate data quality
+
+To set rights:
+1. Go to `System`  
+2. Click on `Roles`
+3. Select a `Role`
+4. Click on `Permissions`
+4. Click on `Data quality insights`
+5. Choose the status for `Edit Data quality insights activation`
+6. Click on `Save`
+
+![rights on data quality](../img/rights-on-data-quality.png)
```

Comment:
I can't see the image in this PR, is it normal?

Reply:
yes, it will come later :-)